engine/common/net_encode.c

```diff
@@ -473,13 +473,13 @@ qboolean Delta_AddField( const char *pStructName, const char *pName, int flags,
 
 void Delta_WriteTableField( sizebuf_t *msg, int tableIndex, const delta_t *pField )
 {
-	int		nameIndex;
-	delta_info_t	*dt;
+	int nameIndex;
```

Comment:
Very useful changes. :)

Please set up your editor.

Reply:
It uses auto-correction according to clang-format xD
But it's not suitable in cases like this

Reply:
It's overall not really suitable to Xash codebase to be honest. I wrote it years ago and haven't touched it since.

Reply:
Ok I'll revert it